release/charts/chart_management.go

```diff
@@ -21,15 +21,20 @@ type asset struct {
 
 // ChartArgs will return the list of available charts in the current branch
 func ChartArgs(ctx context.Context, c *config.ChartsRelease) ([]string, error) {
-	assets := filepath.Join(c.Workspace, "assets")
+	var (
+		assets string
+		files  []os.DirEntry
+		dirs   []string
+		err    error
```

Comment:
Generally speaking, in Go, pre declaring any err var is a code smell and shouldn't be done.